web/index.html

```diff
@@ -34,11 +34,11 @@
         }
         :focus-visible {
             outline: 0;
-            box-shadow: 0px 0px 0px 1px #0185ff;
+            box-shadow: inset 0px 0px 0px 1px #0185ff;
```

Comment:
Rather than using a hex code, can we refer to a color variable instead please?

Reply:
Unfortunately I believe this suffers from the same issue we're encountering with trying to select specific components for inside/outside shadows: this is outside the scope of the main javascript source code.

So I'm not immediately sure of a way to reference a variable in `colors.js` in this pseudo selector. @Santhosh-Sellavel @puneetlath Any ideas?